docs/syntax.rst

```diff
@@ -160,16 +159,26 @@ Selectors
         "unreadEmails": 5
     }
 
-One of the most common cases when a localizer needs to use a placeable is when 
-there are multiple variants of the string that depend on some external 
-argument.
-
-FTL provides a select expression that chooses one of the provided variants 
-based on the given selector.
-
-By default, when a number is used as a selector, FTL implicitly uses ``PLURAL`` 
-formatter that selects the proper plural case for a given language. In English 
-it will be either ``one`` or ``other``.
+One of the most common cases when a localizer needs to use a placeable is when
+there are multiple variants of the string that depend on some external
+argument.  FTL provides the select expression syntax, which chooses one of the
+provided variants based on the given selector.
+
+The selector may be a string in which case it will be compared directly to the
+keys of variants defined in the select expression.  Or it can be a number, like
+in the example above.  FTL will implicitly convert it into the proper plural
+case for a given language. In English it will be either ``one`` or ``other``.
```

Comment:
I actually think we can be better on the "implicitly convert". Something like:

For numbers, the variants either match numbers exactly, or plural form names.

Not really happy with the nomenclature here, but I think we should make clear that it's both, and not just one or other.

Also, should we make it explicit that this is CLDR, and what the maximum set of options are?